README.md

```diff
@@ -1,11 +1,54 @@
-## My Project
+## ACK Multi-Controller Helm Chart
 
-TODO: Fill this README out!
+The ACK Multi-Controller Helm chart, shortened to `ack-chart`, acts as a wrapper
+for all ACK controllers released under General Availability (GA). 
```

Comment:
Nit: include here a link to the "canonical list" of GA controllers as published on the public docs site